src/olympia/api/serializers.py

```diff
@@ -40,9 +41,15 @@ def get_fields(self):
                 fields[key] = ESTranslationSerializerField(source=field.source)
         return fields
 
-    def to_native(self, data):
+    def to_representation(self, data):
+        # We are passing search results (`Result` instances) directly
+        # to the serializer we support this here rather than transforming
+        # it there every time.
```

Comment:
:+1: on the comment but this is confusing to read. (`to the serializer we support this here`; typo somewhere ?)